Dockerfile

```diff
@@ -28,6 +28,10 @@ RUN apt-get update -qq && \
 # http://unix.stackexchange.com/questions/195975/cannot-force-remove-directory-in-docker-build
 #        && rm -rf /var/lib/apt/lists
 
+# adding custom locales to provide backward support with scrapy cloud 1.0
+COPY locales /usr/local/share/
```

Comment:
why not replace `/etc/locale.gen` completely? it's either way has all content commented out by default

```
COPY locales /etc/locale.gen
RUN locale-gen
```